exporter/datadogexporter/internal/hostmetadata/internal/ec2/ec2.go

```diff
@@ -64,41 +63,23 @@ func GetHostInfo(logger *zap.Logger) (hostInfo *HostInfo) {
 		return
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 2*override.TimePerCall)
-	defer cancel()
+	meta := ec2metadata.New(sess)
```

Comment:
Reverted this back to the original state before my changes. @adam-mateen request to not change datadog.

Reply:
Could not revert datadog go.mod/sum bc it takes in the override from the connector